doc/nrf/dev_model_and_contributions/contributions.rst

```diff
@@ -0,0 +1,56 @@
+.. _contributions_ncs:
+
+Contributing to the |NCS|
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Any individual or company can choose to contribute to the |NCS| codebase.
```

Comment:
I think we should make it clear here that, if the issue is in Zephyr code, it is much preferrable to contribute it directly upstream to the Zephyr project.

Reply:
Added a note.